app/services/organizations/importers/google_csv_import_service.rb

```diff
@@ -3,41 +3,61 @@
 module Organizations
   module Importers
     class GoogleCsvImportService
+      Status = Data.define(:success?, :count, :no_match, :errors)
       def initialize(file)
         @file = file
         @organization = Current.organization
+        @count = 0
+        @no_match = []
+        @errors = []
       end
 
       def call
+        row_num = 1
```

Comment:
do we not have access to a row number helper method with this gem?

Reply:
I couldn't find it in the CSV doc's. Would love to be told I'm wrong haha

Reply:
Monsieur GPT said
The Ruby CSV library does not provide a direct row or row_number helper method for tracking the row number during iteration. However, you can easily track row numbers yourself while iterating through a CSV file by using with_index.

Here’s how you can get the row number with each row in Ruby:

```
CSV.foreach("your_file.csv", headers: true).with_index(1) do |row, index|
  puts "Row #{index}: #{row}"
end
```

Reply:
Do you feel that is cleaner that what I currently have? Happy to change it, or change the naming of `row_num` to `index`. Not sure `with_index` gains anything.

Reply:
I think with_index is a little cleaner. I think generally if index methods are available it makes sense to use them. So I’ve been told on my PR reviews at work :)

Reply:
That does look better now that I typed it out. Thanks!!